gradle/publish-android-lib.gradle

```diff
@@ -118,9 +119,7 @@ afterEvaluate { project ->
     task androidJavadocs(type: Javadoc) {
         source = android.sourceSets.main.java.srcDirs
 
-        classpath += project.files(android.getBootClasspath())
-        classpath += configurations.compileJavadoc
-        classpath += configurations.compile + configurations.provided
+        classpath += project.android.libraryVariants.toList().first().javaCompile.classpath
```

Comment:
When it uses `configuarations`, It fails to build becuase there's no `debug` or `release` things. (`failOnBuild false` doesn't work.)

Reply:
Maybe the entire task should be deleted? Although if it works like this, I don't mind.

Reply:
Done in 2e13fc2